src/renderer/src/components/common/Series/SeriesSidebar.tsx

```
@@ -0,0 +1,60 @@
+import { flex } from '@renderer/utils'
+import { styled } from 'styled-components'
+import SeriesRoot from './SeriesRoot/SeriesRoot'
+import { useEffect, useState } from 'react'
+import { TreeNode } from '@renderer/types/series/type'
+import SeriesFolder from './SeriesFolder/SeriesFolder'
+import SeriesFile from './SeriesFile/SeriesFile'
+import { useSeriesStore } from '@renderer/stores/seriesStore'
+
+const SeriesSidebar = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)